02_classify_stratified/nbconverted/plot_purity_results.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Plot tumor purity prediction results
+# 
+# Here, we'll visualize the results of our tumor purity prediction. For now, we're trying to predict whether a given sample is above or below the median tumor purity (i.e. classification).
```

Comment:
Did you try a regression model on the continuous purity value? Just wondering, might not make sense.

Reply:
Not yet - that's our next experiment!